magstock/templates/registration/checkin.html

```diff
@@ -0,0 +1,154 @@
+<script type="text/javascript">
+    $(function() {
+        $('.dialog-form').dialog({ autoOpen: false });
+
+        $('.attendee-checkin').button().on('click', function () {
+            var attendee_id = $(this).attr('data-attendeeid');
+            var dialog = $('#' + attendee_id).dialog({
+                autoOpen: false,
+                height: 450,
+                width: 500,
+                modal: true,
+                buttons: [{
+                    text: 'Save & Check In',
+                    click: function() {
+                        check_in(attendee_id);
+                        dialog.dialog('close');
+                    }
+                }, {
+                    text: 'Cancel',
+                    click: function () {
+                        dialog.dialog('close');
+                    }
+                }],
+                // FIXME: OMG TERRIBLE POSITION HAX
+                open: function () {
+                  $('.ui-dialog').css('position', 'fixed')
+                    .css('top', '5%')
+                    .css('left', '33%');
+                }
+            });
+            dialog.find('input.num').keypress(function (event) {
+                if (event.keyCode === 13) {  // Enter key
+                    event.preventDefault();
+                    dialog.dialog('close');
+                }
+            });
+            dialog.dialog('open');
+            dialog.dialog('option', 'position', {at: 'center', of: window});
+        });
+    });
+
+    var check_in = function (id) {
+        if ($('#age_' + id).val() == {{ c.AGE_UNKNOWN }}) {
+            alert('You may not check someone in without confirming their age.');
+            return;
+        }
```

Comment:
This is what I changed; I pull the new form fields (license plate and site number) and submit them to the new ajax handler, and then submit to the regular check-in page after that succeeds.